docs/ref/forms/models.txt

```diff
@@ -1,15 +1,257 @@
+
+.. _ref-modelform-meta-options:
+
+ModelForm ``Meta`` Options
+==========================
+
+The :class:`~django.forms.ModelForm` helper class can be used to build forms
+that closely reflect specific Django models. The exact structure of the form
+produced can be controlled by setting metadata options as attributes of an inner
+``Meta`` class.
+
+::
+
+    # Example using some sort of Book model
+
+    from django.forms import ModelForm
+    from myapp.models import Book
+
+
+    class BookForm(ModelForm):
+        class Meta:
+            model = Book
+            fields = ["title", "author"]
+            help_texts = {
+                "title": "The title of the book",
+                "author": "The author of the book",
+            }
+            # ... other attributes
+
+.. admonition:: Invalid Meta class attributes
+
+    While any number of attributes can be included in a ``Meta``
+    class, **only attributes that match those described below will be used**.
+    When troubleshooting or encountering unexpected behavior, be sure to check
+    for typos or misnamed attributes because these will **not**, in and of
+    themselves, raise an error (though they might cause another error to be
+    raised if they affect a required attribute).
+
+:attr:`~ModelFormOptions.model` is required as well as either 
+:attr:`~ModelFormOptions.fields` or :attr:`~ModelFormOptions.exclude`.
+All other ``Meta`` attributes are optional. Included fields need not be 
+included in every option used. If an option isn't set, or if a model's 
+field isn't used with a particular option, Django will fall back on whatever 
+has already been defined for a field in their model or whatever defaults 
+might exist for that field's type.
+
+Optional ``Meta`` attributes, other than
+:attr:`~ModelFormOptions.localized_fields` and
+:attr:`~ModelFormOptions.formfield_callback`, expect a dictionary that maps a
+model field name string to some other object or string. Including invalid or
+excluded field name strings in these dictionaries is possible but should have no
+effect since fields that haven't been included would never be accessed.
+
+``error_messages``
+------------------
+
+.. attribute:: ModelFormOptions.error_messages
+
+    Expects a dictionary that maps a model field name string to a dictionary of
+    error message keys (``null``, ``blank``, ``invalid``, ``unique``, etc.)
+    mapped to custom error messages.  
+    
+    Where a field doesn't have an error message dictionary mapped to it in
+    :attr:`~ModelFormOptions.error_messages`, Django will fall back on error
+    messages defined in that model field's
+    :attr:`django.db.models.Field.error_messages` and then finally on the
+    default error messages for that field type.
+
+``exclude``
+-----------
+
+.. attribute:: ModelFormOptions.exclude
+
+    *One of* :attr:`~ModelFormOptions.fields` *and*
+    :attr:`~ModelFormOptions.exclude` *must be set.* Expects a tuple of strings
```

Comment:
Note that a single * is in italics. Not sure if that's on purpose?
I also feel like we can say this last and first say what it is :thinking: